common/Common.hpp

```diff
@@ -37,6 +37,7 @@ constexpr int COMMAND_TIMEOUT_MS = COMMAND_TIMEOUT_SECS * 1000;
 constexpr int CHILD_TIMEOUT_MS = COMMAND_TIMEOUT_MS;
 constexpr int CHILD_REBALANCE_INTERVAL_MS = CHILD_TIMEOUT_MS / 10;
 constexpr int POLL_TIMEOUT_MICRO_S = (COMMAND_TIMEOUT_MS / 5) * 1000;
+constexpr int POLL_FORKIT_TIMEOUT_SECS = 60;
```

Comment:
Note for the future: We should use `std::chrono` directly here for all these and avoid all the conversions in the code.

Reply:
Yep - ideally we should have these as global variables that can be tweaked too - but that creates a load of linking / stubbing problems for different impls I think.